### DISCUSSION TITLE
next js guide update

### DISCUSSION THREAD

**Discussion by LoykDago** [mantinedev/mantine#4791] [Feature requests]
Hello  , sorry in advance for the bad english 

I was using mantine js with next js but next js add some updates with the new app router . So could you please update the page about the next js guide for the newcomers . I had some issue with the server side rendering and i skip the guide because it is using the old next js pages router system which is my mistake because the solution was there . To avoid more frustration for those who will make a project like i did , can you add full guide  to setup the SSR with mantine and the new app router of next js .

Thank you

**Comment by rtivital** [comment]
Starting from v7 Mantine is fully compatible with Next.js app router – https://mantine.dev/guides/next
6.x versions support only pages router.
